### DISCUSSION TITLE
VULNERABILITY: inflight Missing Release of Resource after Effective Lifetime

### DISCUSSION THREAD

**Discussion by shijumon** [exceljs/exceljs#2668] [General]
Snyk vulnarabilty scan reporting a vulnarabilty in the excel js library
`VULNERABILITY: inflight Missing Release of Resource after Effective Lifetime`

Overview
Affected versions of this package are vulnerable to Missing Release of Resource after Effective Lifetime via the makeres function due to improperly deleting keys from the reqs object after execution of callbacks. This behavior causes the keys to remain in the reqs object, which leads to resource exhaustion.

Exploiting this vulnerability results in crashing the node process or in the application crash.

Is there any fix coming?

**Comment by AlisherAmonulloev** [comment]
plus one here
I also posted an issue: https://github.com/exceljs/exceljs/issues/2715#issue-2162923288
